src/routes/cart.router.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  addArticleToCart,
+  clearCart,
+  createCart,
+  getCart,
+  quantityArticleCart,
+  removeArticleFromCart,
+} from '@controlers/cart.controller';
+import { isAuth } from '@utils/isAuth';
+import { Router } from 'express';
+
+const cartRouter = Router();
+
+cartRouter.get('/cart', isAuth, getCart);
+cartRouter.post('/cart', isAuth, createCart);
```

Comment:
why is there event to create cart? user dont need this option because it should be automatically done when he add first product to cart.